src/librustc/mir/mod.rs

```diff
@@ -1149,11 +1149,17 @@ impl<'tcx> TerminatorKind<'tcx> {
             Return | Resume | Abort | Unreachable | GeneratorDrop => vec![],
             Goto { .. } => vec!["".into()],
             SwitchInt { ref values, switch_ty, .. } => {
+                let size = ty::tls::with(|tcx| {
```

Comment:
Please call this `bits`, or, better, keep the name but do `size.bits() as u8` at use sites.

Reply:
done